pkg/cmd/types.go

```diff
@@ -57,5 +68,39 @@ func (i *ImageInspectorOptions) Validate() error {
 	if len(i.Serve) == 0 && i.Chroot {
 		return fmt.Errorf("Change root can be used only when serving the image through webdav")
 	}
+	if len(i.ScanType) > 0 && len(i.ScanResultsDir) == 0 {
+		return fmt.Errorf("scan-result-dir must be specified with scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 && len(i.ScanType) == 0 {
+		return fmt.Errorf("scan-result-dir can be used only when spacifing scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 {
+		fi, err := os.Stat(i.ScanResultsDir)
+		if err != nil || os.IsNotExist(err) {
```

Comment:
@simon3z The user should create the directory. I didn't but I would use the /tmp directory for that as ManageIQ uses image-inspector-client to read the data from the server anyway and doesn't care about the exact location of this file.
This is what I use for testing:
```sudo ./image-inspector --image=registry.access.redhat.com/openshift3/metrics-cassandra --path=/tmp/image-content --chroot --serve=0.0.0.0:5555 --scan-type=openscap --scan-results-dir=/tmp```